pkg/parser.go

```diff
@@ -172,6 +172,26 @@ func parseStringValue(value interface{}, isNullable bool) Value {
 	}
 }
 
+// parseMapValue parses a map value to JSON.
+func parseMapValue(value interface{}, isNullable bool) Value {
+	// Check if the value is a map
+	switch m := value.(type) {
+	case map[string]interface{}: // Check if it's a map with string keys and any type of value
```

Comment:
Unfortunately, Map can contain any type (Int, Float) as key ... 
did you check how it will serialize to FORMAT JSON response?